packages/fluent/scss/expansion-panel/_layout.scss

```diff
@@ -0,0 +1,66 @@
+@use "_variables.scss" as *;
+@use "../core/color-system" as *;
+
+@mixin kendo-expansion-panel--layout() {
+
+    // Base
+    .k-expander {
+        border-width: var( --kendo-expander-border-width, #{$kendo-expander-border-width} );
+        border-style: solid;
+        box-sizing: border-box;
+        overflow: hidden;
+        outline: 0;
+        font-family: var( --kendo-expander-font-family, #{$kendo-expander-font-family} );
+        font-size: var( --kendo-expander-font-size, #{$kendo-expander-font-size} );
+        line-height: var( --kendo-expander-line-height, #{$kendo-expander-line-height} );
+        display: flex;
+        flex-direction: column;
+        -webkit-touch-callout: none;
+        -webkit-tap-highlight-color: $rgba-transparent;
+
+        &:not(:last-child) {
+            &:not(.k-focus) {
+                margin-block-end: var( --kendo-expander-margin-y, #{$kendo-expander-margin-y} );
+            }
+        }
```

Comment:
why do we need this margin ?

Reply:
I was looking for a way to remove one of the horizontal shadows when the panels are in a group. There is a different solution in the next commit.